src/librustdoc/html/static/storage.js

```diff
@@ -1,6 +1,5 @@
 // From rust:
 /* global resourcesSuffix */
-
```

Comment:
This unrelated whitespace deletion accidentally snuck in in my branch. Sorry about that. Would be good to revert it to avoid spurious diffs.